cypress/integration/articleFlows/commentOnArticle.spec.js

```diff
@@ -402,3 +404,15 @@ describe('Comment on articles', () => {
     });
   });
 });
+
+const getCommentCombobox = () =>
```

Comment:
Documentation (blocking): Leave a comment in the code as to why we are not using aliases and these functions instead.

nitpick (non-blocking): Move these functions to the top of the file within the first describe.